
(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:177:16: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:180:16: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:553:32: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:563:32: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:572:48: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:703:7: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:717:7: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:746:7: Missing name of pseudo-class

(sugar-activity:2381): Gtk-WARNING **: Theme parsing error: gtk-widgets.css:760:7: Missing name of pseudo-class
1445040484.994741 WARNING root: No Gtk.AccelGroup in the top level window.

(sugar-activity:2381): Gdk-CRITICAL **: gdk_device_get_source: assertion `GDK_IS_DEVICE (device)' failed

(sugar-activity:2381): Gdk-CRITICAL **: gdk_device_get_source: assertion `GDK_IS_DEVICE (device)' failed

(sugar-activity:2381): Gdk-CRITICAL **: gdk_device_get_source: assertion `GDK_IS_DEVICE (device)' failed
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
1445042544.610455 ERROR dbus.connection: Exception in handler for D-Bus signal:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/dbus/connection.py", line 230, in maybe_handle_message
    self._handler(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/sugar3/presence/connectionmanager.py", line 91, in __status_changed_cb
    self._connections_per_account[account_path].connected = False
KeyError: dbus.ObjectPath('/org/freedesktop/Telepathy/Account/salut/local_xmpp/account0')
1445043490.594839 WARNING root: No Gtk.AccelGroup in the top level window.
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/dbus/connection.py", line 604, in msg_reply_handler
    reply_handler(*message.get_args_list(**get_args_opts))
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 688, in __save_cb
    self._complete_close()
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 1016, in _complete_close
    self.shared_activity.leave()
  File "/usr/lib/python2.7/site-packages/sugar3/presence/activity.py", line 474, in leave
    self.telepathy_text_chan.Close()
  File "/usr/lib/python2.7/site-packages/dbus/proxies.py", line 145, in __call__
    **keywords)
  File "/usr/lib/python2.7/site-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name :1.36 was not provided by any .service files
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/gi/_gobject/__init__.py", line 266, in obj_set_property
    prop.fset(self, value)
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 513, in set_active
    self.save()
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 815, in save
    preview = self.get_preview()
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 728, in get_preview
    dummy_cr = Gdk.cairo_create(window)
TypeError: Argument 0 does not allow None as a value
1445049864.851981 ERROR root: Error saving activity object to datastore
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 1001, in _prepare_close
    self.save()
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 815, in save
    preview = self.get_preview()
  File "/usr/lib/python2.7/site-packages/sugar3/activity/activity.py", line 728, in get_preview
    dummy_cr = Gdk.cairo_create(window)
TypeError: Argument 0 does not allow None as a value
g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
